test/testcafe/spec/ReEnrollAuthenticatorWarningPasswordView_spec.js

```diff
@@ -91,7 +91,7 @@ async function setup(t) {
       }
       await t.expect(passwordExpiryWarningPage.remindMeLaterLinkExists()).eql(true);
       await t.expect(passwordExpiryWarningPage.getSignoutLinkText()).eql('Back to sign in');
-      await t.expect(passwordExpiryWarningPage.doesTextExist('When your password expires you will be locked out of your Okta account.')).eql(true);
+      await t.expect(passwordExpiryWarningPage.doesTextExist('When your password expires, you will have to change your password before you can login to your Localhost account.')).eql(true);
```

Comment:
Related to this commit: https://github.com/okta/okta-signin-widget/commit/6e25d801f521577f5f321d8801eac6e37f4c1121